grakn-migration/json/src/main/java/ai/grakn/migration/json/JsonMigrator.java

```diff
@@ -46,28 +46,30 @@
     private final Set<Reader> readers;
 
     public static void main(String[] args) {
-        MigrationCLI.init(args, JsonMigrationOptions::new).stream()
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .forEach(JsonMigrator::runJson);
+        try {
+            MigrationCLI.init(args, JsonMigrationOptions::new).stream()
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .forEach(JsonMigrator::runJson);
+        } catch (Throwable throwable){
+            MigrationCLI.die(throwable.getMessage());
```

Comment:
If it's all or nothing then in my opinion in-line it. Only the user should be aware of a failure in an all or nothing situation. Logs are reserved fro tracking issues which can affect us in the long run. In this case nothing went in so nothing to worry about.